src/Zynga/Framework/IO/Web/V1/Manager.hh

```diff
@@ -50,4 +53,25 @@ class Manager {
       throw new UnexpectedHttpCodeException((string)$returnCode);
     }
   }
+  
+  public static function makeCurlRequest(
+    UrlBox $uploadUrl,
+    Map<string, mixed> $postParams): array<string, mixed> {
+      if(self::$useMockCurl == false) {
+        $session = curl_init();
+        curl_setopt( $session, CURLOPT_URL, $uploadUrl->get());
+        curl_setopt( $session, CURLOPT_POST, true);
+        curl_setopt( $session, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt( $session, CURLOPT_POSTFIELDS, http_build_query($postParams));
+        curl_setopt( $session, CURLOPT_TIMEOUT, 5);
```

Comment:
We should let the timeout be defined by the calling code as this won't be a good general case.

For example, timeouts in Apache can be configured to way less than this.

Reply:
Thats a good point, there is no point setting this actually.